dashboard/app/models/pd/workshop.rb

```diff
@@ -81,6 +81,7 @@ class Pd::Workshop < ActiveRecord::Base
       SUBJECT_CS_IN_S_PHASE_3_SEMESTER_2 = 'Phase 3 - Semester 2'
     ],
     COURSE_CSP => [
+      SUBJECT_CSP_SUMMER_WORKSHOP = 'Summer Workshop',
```

Comment:
I don't think it's a "big problem", but these are typically listed in the order they're meant to be taken. It would be strange to list Workshop2 before Workshop1, and similarly if the summer workshop occurs after workshops 1-4 (as I think it does though I'm not sure), it would be strange to list it first. Anyway it's easy to change later after we hear from the PLC team.